src/tests.cpp

```diff
@@ -0,0 +1,85 @@
+#include "quadratic_equation.h"
+
+struct CoefficentsForLinear {

```

Comment:
Are these all really coefficients (oh, yeah, there's also a typo here ! PLEASE USE A SPELL-CHECKER !) of an equation? Rename it.